addons/mail/static/src/core/common/message_model.js

```diff
@@ -478,6 +478,22 @@ export class Message extends Record {
         }
     }
 
+    enterEditMode(thread) {
```

Comment:
`thread` looks unused?

Reply:
This is used to remember the thread in which the message edition has started, so that when the edition has stopped, it recovers the focus on this thread (as long as this is still the thread being viewed).

Note that this is not necessarily the origin thread of message, e.g. can edit message from Inbox / Starred / History.